src/main/java/de/craften/plugins/rpgplus/components/ability/Ability.java

```diff
@@ -0,0 +1,13 @@
+package de.craften.plugins.rpgplus.components.ability;
+
+import org.bukkit.entity.Player;
+
+public interface Ability {
+
+	String getIdentifier();
+	String getDisplayName();
+	default void giveTo(Player player) { }

```

Comment:
Why empty default implementations for `giveTo` and `removeFrom`? An ability without these two methods doesn't make sense at all. :smile: